sdk/storage/Azure.Storage.sln

```diff
@@ -0,0 +1,131 @@
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution", "Solution", "{A4D0A040-DB1C-4B1D-B4D8-8004C58DB49D}"
+	ProjectSection(SolutionItems) = preProject
+		.editorconfig = .editorconfig
+		Directory.Build.props = Directory.Build.props
+		generate.ps1 = generate.ps1
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Blobs", "Blobs", "{B93D17A7-F7D7-4A2E-9E57-7B6ED57EF749}"
+	ProjectSection(SolutionItems) = preProject
+		Azure.Storage.Blobs\README.md = Azure.Storage.Blobs\README.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Queues", "Queues", "{F4218B06-A246-4762-984D-5BC852B62889}"
+	ProjectSection(SolutionItems) = preProject
+		Azure.Storage.Queues\README.md = Azure.Storage.Queues\README.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Files", "Files", "{9EF57C69-DC48-4E5C-BBD5-76E23E4ADDA6}"
+	ProjectSection(SolutionItems) = preProject
+		Azure.Storage.Files\README.md = Azure.Storage.Files\README.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Common", "Common", "{794500AE-0D91-414D-8C8C-08577C65AA7A}"
+	ProjectSection(SolutionItems) = preProject
+		Azure.Storage.Common\README.md = Azure.Storage.Common\README.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Swagger", "Swagger", "{69A8687B-B040-451E-A72E-295C6CFC1B8B}"
+	ProjectSection(SolutionItems) = preProject
+		Swagger\Blobs\blob.json = Swagger\Blobs\blob.json
+		Swagger\Blobs\readme.md = Swagger\Blobs\readme.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Swagger", "Swagger", "{AD85524A-FB82-475C-9907-27DBC2BE2945}"
+	ProjectSection(SolutionItems) = preProject
+		Swagger\Files\file.json = Swagger\Files\file.json
+		Swagger\Files\readme.md = Swagger\Files\readme.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Swagger", "Swagger", "{6EEF892F-E4EE-4610-9C94-4B206D9E7152}"
+	ProjectSection(SolutionItems) = preProject
+		Swagger\Queues\queue.json = Swagger\Queues\queue.json
+		Swagger\Queues\readme.md = Swagger\Queues\readme.md
+	EndProjectSection
+EndProject
+Global
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		Azure.Storage.Shared\tests\Azure.Storage.Tests.Shared.projitems*{1dfd6176-906a-4620-951b-9ab0ad619fa8}*SharedItemsImports = 13
+		Azure.Storage.Shared\src\Azure.Storage.Shared.projitems*{637d8341-246c-4f63-bc11-820035bf1b32}*SharedItemsImports = 13
+	EndGlobalSection
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{9056EE3A-9ED4-441A-9A88-A567ED07DDDD}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
I recall seeing mention of Preview builds in some of the other projects already in the repo?

Reply:
I don't know if any of the track 2 libraries have a Preview configuration, but if you check out https://github.com/Azure/azure-sdk-for-net/tree/master/eng/ you can see the work Wes is doing to create a daily feed.